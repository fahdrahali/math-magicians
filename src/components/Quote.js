import React from 'react';

const Quote = () => (
  <p className="quote">
    Mathematics is not about numbers, equations, computations, or algorithms: it is about
    understanding. -
    <span> William Paul Thurston</span>
  </p>
);

export default Quote;
