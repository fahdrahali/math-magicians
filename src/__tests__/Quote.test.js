import React from 'react';
import TestRenderer from 'react-test-renderer';
import Quote from '../components/Quote';

it('renders correctly', () => {
  const renderer = TestRenderer.create(<Quote />).toJSON();
  expect(renderer).toMatchSnapshot();
});
