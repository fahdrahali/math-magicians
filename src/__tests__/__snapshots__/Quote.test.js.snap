// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p
  className="quote"
>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. -
  <span>
     William Paul Thurston
  </span>
</p>
`;
